dtmi/ip3/hrm-1.json

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:ip3:hrm;1",
+  "@type": "Interface",
+  "displayName": "Smart Electric Meter",
+  "description": "Ip3.co.ltd",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "sn",
+      "displayName": "Device SN",
+      "description": "Device Serials Number.ex S202104150007.",
+      "schema": "string"
+    },
+    {
+      "@type": "Property",
+      "name": "sw",
+      "displayName": "Software Version",
+      "description": "The device software version,ex.sw:NB.1.1.2.",
+      "schema": "string"
+    },
+    {
+      "@type": "Property",
+      "name": "teleFrq",
+      "schema": "double",
```

Comment:
Are you OK with this property missing a `displayName`?

Reply:
OK,you can do that.